src/commands/schema/authorize.ts

```diff
@@ -0,0 +1,47 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { addSchemaPermission } from '../../lib/aws-util.js'
+import { lambdaAuthBuilder, type LambdaAuthFlags } from '../../lib/command/common-flags.js'
+import {
+	smartThingsCommand,
+	smartThingsCommandBuilder,
+	type SmartThingsCommandFlags,
+} from '../../lib/command/smartthings-command.js'
+
+
+export type CommandArgs =
+	& SmartThingsCommandFlags
+	& LambdaAuthFlags & {
+		arn?: string
+	}
+
+const command = 'schema:authorize <arn>'
+
+const describe = 'authorize calls to your schema app AWS Lambda function from SmartThings'
```

Comment:
Should "schema app" be capitalized, here and presumably elsewhere?

Reply:
They should be all capitalized now.